router/router_test.go

```diff
@@ -481,18 +483,58 @@ func TestRouter(t *testing.T) {
 				So(len(babbageHandler.ServeHTTPCalls()), ShouldEqual, 0)
 			})
 		})
-		Convey("When a related data request is made, and the RelatedDataRouteEnabled is enabled", func() {
+		Convey("When a related data request is made, the RelatedDataRouteEnabled is enabled and the Legacy Cache Proxy is disabled", func() {
 			url := "/economy/relateddata"
 			req := httptest.NewRequest("GET", url, http.NoBody)
 			res := httptest.NewRecorder()
 
 			config.RelatedDataRouteEnabled = true
 			config.SearchRoutesEnabled = true
+			config.LegacyCacheProxyEnabled = false
+
 			r := router.New(config)
 			r.ServeHTTP(res, req)
-			Convey("Then no requests are sent to Zebedee", func() {
-				So(len(zebedeeClient.GetWithHeadersCalls()), ShouldEqual, 0)
+			Convey("Then the request is sent to the search handler", func() {
+				So(len(searchHandler.ServeHTTPCalls()), ShouldEqual, 1)
```

Comment:
I prefer using the inbuilt func for measuring length 
```suggestion
				So(searchHandler.ServeHTTPCalls(), ShouldHaveLength, 1)
```

Reply:
Terrible copypasta